omero/sysadmins/omeroweb-upgrade.rst

```diff
@@ -121,11 +121,11 @@ The process for clearing the session store depends on the storage backend:
   used as the prefix of the file sessions and the command above needs
   to be modified accordingly.
 
-- OMERO.web session data is stored using the `Redis <https://redis.io/>`_ store
+- OMERO.web sessions are stored using the `Redis <https://redis.io/>`_ store
   if the :property:`omero.web.session_engine` property is set to
   `django.contrib.sessions.backends.cache` and Redis is configured via the
   :property:`omero.web.caches` property by setting the `BACKEND` to
-  'django.core.cache.backends.redis.RedisCache' and `LOCATION` to
+  `django.core.cache.backends.redis.RedisCache` and `LOCATION` to
   the URL of the Redis instance.
 
   Session data is stored as key/value pairs in the Redis store with the name of
```

Comment:
``Sessions are stored as key/value`` should also be adjusted to ``Sessions are..`` in that case.
Same for ``Session data is stored under a temporary folder```